api/medical/views.py

```diff
@@ -204,18 +204,18 @@ def get(self, request):
 
     def post(self, request):
         id = request.POST.get("id", None)
-        accepted = request.POST.get("accepted", None)
+        status = request.POST.get("status", None)
         appointment = Appointment.objects.filter(id=id).first()
-        if accepted == "true":
+        if status == 2:
```

Comment:
USE a constant variable such as
```python
APPOINTMENT_STATUS = {
      'REJECTED': 0,
      'IN_PROGRES': 1,
      'APPROVED': 2
}

```

Reply:
the constant variable is used in the model this is only for the view

Reply:
Share the same constant in both models.py and views.py

Reply:
done